slang.md

```diff
@@ -13,3 +13,9 @@ Former functional human beings that suffered from the Nueroinfinitus (forevermor
 ### Forevermore Plague
 
 Nueroinfinitus also known as the Forevermore Plague was a widedspread virus that effected nearly all persons who had cyber implants
+
+### The Road
+The YANG LIWEI INTERNATIONAL SPACE LIFT. nothing on earth has caused a larger technological, scientific, economic, or industrial boom. Some say "The Road" is the apex achievement of humankind.
+
+### E-Sheks / Joe-Bucks / Script
```

Comment:
Joe Buck? Was this a real person? @The-Real-Matthew-Dycus

Reply:
yes! Joseph Stalin :) its the new ruble

Reply:
yes, joe bucks are based on joseph stalin ,